### ISSUE TITLE
[Godot 4.0] Unable to change root type of imported model to Node3D

### ISSUE THREAD

**Issue by Kuma-Boo** [godotengine/godot#73044]
### Godot version

v4.0.rc1.mono.official [8843d9ad3]

### System information

Windows 10

### Issue description

I noticed this when importing some models, but the importer still uses ```Spatial``` as the root 3D node type.

![image](https://user-images.githubusercontent.com/46894189/218130385-a8b325fe-b702-478e-9096-f041791c891a.png)

```Spatial``` doesn't exist in the search bar since it was renamed to ```Node3D``` a while ago.
~~However, due to the importer expecting ```Spatial```, it's impossible to revert the root type to ```Node3D```.~~
*Edit: Upon further testing, the two don't have anything to do with each other.*

https://user-images.githubusercontent.com/46894189/218132855-c1e21469-1da7-4fcf-9872-f0681ba24fa5.mp4

### Steps to reproduce

1. Import any 3D model.
2. Attempt to change the root type to ```Node3D```.

### Minimal reproduction project

N/A

***EDIT***
It appears that being ```Spatial``` is due to the project originally being from before the ```Spatial``` to ```Node3D``` rename occurred. However, the issue of being unable to change the root type back to ```Node3D``` persists.

**Comment by akien-mga** [comment]
That's interesting, I don't find any reference to "spatial" or "Spatial" in relevant code. The default value seems to be "Node3D".

Maybe those are older files which were already imported / had their `.import` files generated before the Spatial -> Node3D rename? Can you reproduce the issue in a new project?

**Comment by Kuma-Boo** [comment]
After some testing on a new project, it only occurs on models that were imported on the 3.x branch, and deleting the individual ```.import``` files forces a re-import of the model and fixes the root type to be Node3D.

But when I tried doing this on my personal project it didn't fix anything, and for some reason even the new models were being imported as a Spatial.

After more digging around, the issue seems to come from this line in ```project.godot```:
```
[importer_defaults]

scene={
...
"nodes/root_type": "Spatial",
...
}
```
I must have upgraded my project to 4.0 before the rename happened, so this wasn't changed. After replacing Spatial with Node3D, things aren't being imported as Spatials anymore! 🎉🎉🎉 

However, the second half of the bug where you can't change the root type to Node3D from the editor still exists. It needs to be done by manually changed by setting ```nodes/root_type="Node3D"``` in the ```.import``` file from an external text editor.

**Updated reproduction steps:**

1. Create a new project
2. Import a 3d model
3. Change the root type to any other type of node (i.e. StaticBody3D)
4. Attempt to change the root type back to Node3D

Video:

https://user-images.githubusercontent.com/46894189/218150704-dcf8f1d4-9ae6-4d02-b90d-be58b6f3d9d2.mp4

**Comment by Kuma-Boo** [comment]
Not sure what changed, but this seems to be fixed as of v4.0.stable.mono.official [92bee43ad].
